src/main/java/com/osiris/autoplug/client/configs/SSHConfig.java

```diff
@@ -58,47 +63,66 @@ public SSHConfig() throws IOException, DuplicateKeyException, YamlReaderExceptio
         enabled = put(name, "enabled").setDefValues("false")
             .setComments(
                 "Enables a SSH console for remote access to AutoPlug.",
-                "You can connect to it via any SSH client like Putty, a custom script, or the built-in SSH client in Windows, Linux distros, MacOS, etc.");
+                "You can connect to it via any SSH client like Putty, Terminus, a custom script, or the built-in SSH client in Windows, Linux distros, MacOS, etc.");
 
         port = put(name, "port").setDefValues("22")
             .setComments(
                 "The port the SSH console listens on.",
-                "The default port is 22. Change it if you have a different port setup in your network, are hosting one or more other SSH-based services on the same port, or are otherwise using the default port for something else.");
+                "The default port is 22. Change it if you have a different port setup in your network, are hosting one or more other services on the same port, or are otherwise wanting to use the SSH service on a different port.",
+                "Example:",
```

Comment:
If you have a default value provided already no need to also mention it in the comment and duplicate the info.